collections/_release_4_5/entry-highlights-screen-reader-support.md

```diff
@@ -9,7 +9,7 @@ blockquote: Make your game accessible to visually impaired people
 text: |
   Accessibility should be every developer’s top priority, full-stop. Excluding someone from an experience for factors outside of their control is an area that video games and applications have the potential to circumvent entirely.
 
-  One important feature to support in computer software are screen readers. They are an essential tool for people who are visually impaired, illiterate, or have a learning disability.[^screen-reader-support-mdn] It enables them to understand the context given visually. It does, however, take a solid framework to develop such accommodations. That’s because each platform has its own way to handle accessibility. This obviously makes it difficult to support every platform.
+  One feature often overlooked that is a must-have in computer software is screen readers support. Such readers are an essential tool for people who are visually impaired, illiterate, or have a learning disability.[^screen-reader-support-mdn] It enables them to understand the context given visually. It does, however, take a solid framework to develop such accommodations. That’s because each platform has its own way to handle accessibility. This obviously makes it difficult to support every platform.
```

Comment:
```suggestion
  One feature often overlooked that is a must-have in computer software is screen reader support. Such readers are an essential tool for people who are visually impaired, illiterate, or have a learning disability.[^screen-reader-support-mdn] It enables them to understand the context given visually. It does, however, take a solid framework to develop such accommodations. That’s because each platform has its own way to handle accessibility. This obviously makes it difficult to support every platform.
```
Singular